docs/book/src/developer/core/tilt.md

```diff
@@ -20,14 +20,64 @@ workflow that offers easy deployments and rapid iterative builds.
 ## Getting started
 
 ### Create a kind cluster
-A script to create a KIND cluster along with a local Docker registry and the correct mounts to run CAPD is included in the hack/ folder.
 
-To create a pre-configured cluster run:
+The following CAPI infrastructure providers are suitable for local development:
+
+- [CAPD](https://github.com/kubernetes-sigs/cluster-api/blob/main/test/infrastructure/docker/README.md) - uses Docker containers as workload cluster nodes
+- [CAPK](https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt) - uses KubeVirt VMs as workload cluster nodes
+
+CAPD is the default as it's more lightweight and requires less setup. KubeVirt is useful when
+Docker isn't suitable for whatever reason.
+
+{{#tabs name:"tab-management-cluster-creation" tabs:"Docker,KubeVirt"}}
+{{#tab Docker}}
+
+To create a kind cluster with CAPD, run the following:
+
+```bash
+make kind-cluster
+```
+
+{{#/tab }}
+{{#tab KubeVirt}}
+
+To create a kind cluster with CAPD, run the following:
 
 ```bash
-./hack/kind-install-for-capd.sh
+make kind-cluster-kubevirt
 ```
 
+<aside class="note">
+
+KubeVirt uses *container disks* to create VMs inside pods. These are special container images which
+need to be pulled from a registry. To support pulling container disks from private registries as
+well as avoid getting rate-limited by Docker Hub (if used), the CAPK script mounts your Docker
+config file inside the kind cluster to let the Kubelet access your credentials.
+
+The script looks for the Docker config file at `$HOME/.docker/config.json` by default. To specify
+a different path, set the following variable before running Make above:
+
+```bash
+export DOCKER_CONFIG_FILE="/foo/config.json"
+```
+
+</aside>
+
+<aside class="note">
+
+The CAPK script needs to figure out your container runtime IP prefix to allow communication with
+workload clusters. The script assumes Docker is used. In case a different runtime is used, specify
+your container runtime's IP prefix manually (the first two octets only):
```

Comment:
Maybe worth adding a short note that it get's expanded to `255.200`-`255.250` in some way?